cypress/e2e/tests/pages/users-and-auth/index.spec.ts

```diff
@@ -74,17 +78,121 @@ describe('Auth Index', { testIsolation: 'off', tags: ['@explorer', '@adminUser']
     userRetentionPo.disableAfterPeriodInput().value().should('equal', '300h');
     userRetentionPo.deleteAfterPeriodCheckbox().isChecked();
     userRetentionPo.deleteAfterPeriodInput().value().should('equal', '600h');
     userRetentionPo.userRetentionCron().value().should('equal', '0 0 1 1 *');
     userRetentionPo.userLastLoginDefault().value().should('equal', '1718744536000');
+    userRetentionPo.disableAfterPeriodCheckbox().set();
+    userRetentionPo.saveButton().expectToBeEnabled();
+    userRetentionPo.saveButton().click();
+    usersPo.waitForPage();
+    usersPo.userRetentionLink().checkVisible();
+    usersPo.userRetentionLink().click();
+    userRetentionPo.deleteAfterPeriodCheckbox().set();
+    userRetentionPo.disableAfterPeriodCheckbox().set();
+    userRetentionPo.saveButton().expectToBeEnabled();
+    userRetentionPo.saveButton().click();
+    usersPo.waitForPage();
+  });
+
+  it('a user account gets blocked after the user retention expires', () => {
+    const runTimestamp = +new Date();
+    const username = `user_to_block_${ runTimestamp }`;
+
+    const usersPo = new UsersPo();
+    const userRetentionPo = new UserRetentionPo();
+
+    cy.createUser({
+      username,
+      globalRole:  { role: 'user' },
+      projectRole: {
+        clusterId:   'local',
+        projectName: 'Default',
+        role:        'project-member',
+      }
+    });
+
+    usersPo.goTo();
+    usersPo.userRetentionLink().click();
+
+    userRetentionPo.waitForPage();
+    userRetentionPo.disableAfterPeriodCheckbox().set();
+    userRetentionPo.disableAfterPeriodInput().set('0.1m');
+    userRetentionPo.userRetentionCron().set('* * * * *');
+    userRetentionPo.userLastLoginDefault().set('1718744536000');
+
+    userRetentionPo.saveButton().expectToBeEnabled();
+    userRetentionPo.saveButton().click();
+    usersPo.waitForPage();
+
+    // logout from admin
+    cy.logout();
+
+    cy.url().should('includes', `${ Cypress.config().baseUrl }/auth/login`);
+
+    // login as test user
+    cy.login(username, Cypress.env('password'));
+
+    // logout from test user
+    cy.logout();
+    cy.url().should('includes', `${ Cypress.config().baseUrl }/auth/login`);
+
+    // login back as admin
+    cy.login();
+
+    usersPo.goTo();
+    usersPo.waitForPage();
+    usersPo.list().resourceTable().sortableTable().rowWithPartialName(username)
+      .accountStatusDisabled()
+      .should('be.visible');
+
+    usersPo.goTo();
+    usersPo.waitForPage();
+    usersPo.list().clickRowActionMenuItem(username, 'Delete');
+    const promptRemove = new PromptRemove();
+
+    promptRemove.confirm(username);
+    promptRemove.remove();
+
+    usersPo.userRetentionLink().click();
+    userRetentionPo.waitForPage();
+    userRetentionPo.disableAfterPeriodCheckbox().set();
+    userRetentionPo.saveButton().expectToBeEnabled();
+    userRetentionPo.saveButton().click();
+    usersPo.waitForPage();
+  });
+
+  it('access to standard user retention page', () => {
```

Comment:
to make it more obvious what we're testing here, lets update to something like "standard user should not have access to user retention page"